script/DeployBase.s.sol

```diff
@@ -170,7 +170,7 @@ abstract contract DeployBase is Script {
             );
 
             verifier = new CSVerifier({
-                locator: address(locator),
+                withdrawTo: locator.withdrawalVault(),
```

Comment:
The name seems confusing to me. Consider renaming to `withdrawalCredentials` or `withdrawalAddress`

Reply:
Fixed.